_docs/sql-reference/sql-commands/021-create-schema.md

```diff
@@ -209,13 +220,23 @@ Values are trimmed when converting to any type, except for varchar.
 
 ## Usage Notes 
 
-### General Information  
-- Schema provisioning only works with tables defined as directories because Drill must have a place to store the schema file. The directory can contain one or more files.  
+### General Information
+- Schema provisioning is supported only for the file system (dfs-based) storage plugins. It works by placing a file `.drill.schema` in the root folder of tables defined as a directory. The directory can contain any number of files (even just one) in addition to the schema file.
```

Comment:
Schema provisioning works only with the file system...

Reply:
Thanks, replaced.